CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+(modification: no type change headlines) and this project adheres to
+[Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [0.0.8] - 2020-05-27
+
+This is a maintenance release with dependency and CI updates together with 
+some coder modernization changes.
```

Comment:
did you mean “code” instead of “coder”?

Reply:
Yes, but "coder" sounds cool as well. 😄

Reply:
(will nevertheless merge, this is such a tiny change)